lms/djangoapps/courseware/views/views.py

```diff
@@ -312,6 +313,7 @@ def get_last_accessed_courseware(course, request, user):
             'request': request,
             'masquerade_user': user,
             'course_id': course_key.to_deprecated_string(),
+            'url_to_enroll': CourseTabView.url_to_enroll(course_key),
```

Comment:
This wasn't intentional. I was just trying to restore it after accidentally deleting it, and I put it back in a different place.